install_config/configuring_openstack.adoc

```diff
@@ -157,3 +157,45 @@ When triggering a containerized installation, only the directories of
 container. Therefore, *_cloud.conf_* should be in *_/etc/origin/_* instead of
 *_/etc/_*.
 ====
+
+[[openstack-applying-configuration-changes]]
+== Applying Configuration Changes
+
+Start or restart {product-title} services on all master and node hosts to apply your
+configuration changes:
+
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-master
+# systemctl restart atomic-openshift-node
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-master
+# systemctl restart origin-node
+----
+endif::[]
+
+Switching from not using a cloud provider to using a cloud provider produces an
+error message. Adding the cloud provider tries to delete the node because the
+node switches from using the *hostname* as the `*externalID*` (which would have
+been the case when no cloud provider was being used) to using the OpenStack
+`*instance-id*` (which is what the OpenStack cloud provider specifies). To resolve
+this issue:
+
+.  Log in to the CLI as a cluster administrator.
+. Check existing node labels:
++
+[source, bash]
+----
+$ oc describe node <node_name> | grep -Poz '(?s)Labels.*\n.*(?=Taints)'
```

Comment:
@qinpingli Can you check if this is fine? If not I will replace it with the suggested `oc get node <node_name> -o yaml` command.